src/modules/mavlink/mavlink_log_handler.cpp

```diff
@@ -109,24 +113,6 @@ MavlinkLogHandler::handle_message(const mavlink_message_t *msg)
 	}
 }
 
-//-------------------------------------------------------------------
-unsigned
-MavlinkLogHandler::get_size()
```

Comment:
Why is this removed?

Reply:
@bkueng, because its simpler to use the data in ::send directly, get_size() function is not really needed

Reply:
I thought you changed the logic, but it's correct.